src/test/java/org/jenkinsci/plugins/blueoceandisplayurl/BlueOceanDisplayURLImplTest.java

```diff
@@ -170,12 +192,18 @@ public void testMultibranchUrls_CustomOrganization() throws Exception {
 
         url = getPath(displayURL.getChangesURL(job.getFirstBuild()));
         Assert.assertEquals("/jenkins/blue/organizations/TestOrg/folder%2Ftest/detail/feature%2Ftest-1/1/changes", url);
+
+        url = getPath(displayURL.getTestsURL(job.getFirstBuild()));
+        Assert.assertEquals("/jenkins/blue/organizations/TestOrg/folder%2Ftest/detail/feature%2Ftest-1/1/tests", url);
+
+        url = getPath(displayURL.getArtifactsURL(job.getFirstBuild()));
+        Assert.assertEquals("/jenkins/blue/organizations/TestOrg/folder%2Ftest/detail/feature%2Ftest-1/1/artifacts", url);
     }
 
     MockFolder orgFolder;
     @Before
     public void setUp() throws IOException {
-        displayURL = Iterables.find(DisplayURLProvider.all(), Predicates.instanceOf(BlueOceanDisplayURLImpl.class));
+        displayURL = new BlueOceanDisplayURLImpl(); //Iterables.find(DisplayURLProvider.all(), Predicates.instanceOf(BlueOceanDisplayURLImpl.class));
```

Comment:
Why was this change made? I think the old code confirms the extension point code works right.

Reply:
Unfortunately the new methods:
- https://github.com/jenkinsci/blueocean-display-url-plugin/pull/22/files#diff-08020ec89303376ad6dd9c3dc5a7a442R97
- https://github.com/jenkinsci/blueocean-display-url-plugin/pull/22/files#diff-08020ec89303376ad6dd9c3dc5a7a442R108

are not exposed within the https://github.com/jenkinsci/display-url-api-plugin/blob/master/src/main/java/org/jenkinsci/plugins/displayurlapi/DisplayURLProvider.java#L22

That's the reason I did not use the `DisplayURLProvider.all`

Reply:
As far as I can tell in the change, displayURL isn't used in your new tests are they?

Reply:
OH YOU ARE! sorry! sorry! i read too fast earlier.